src/Client/ConnectionParameters.h

```diff
@@ -30,7 +31,7 @@ struct ConnectionParameters
     ConnectionParameters(const Poco::Util::AbstractConfiguration & config, std::string host);
     ConnectionParameters(const Poco::Util::AbstractConfiguration & config, std::string host, std::optional<UInt16> port);
 
-    static UInt16 getPortFromConfig(const Poco::Util::AbstractConfiguration & config, std::string connection_host);
+    static UInt16 getPortFromConfig(const Poco::Util::AbstractConfiguration & config, String connection_host);
```

Comment:
Do we need to copy it? (use `const String &` or `std::move`)

Reply:
Can do, but I didn't even change the signature, just used a type alias.